lib/callbacks.js

```diff
@@ -48,46 +40,42 @@ const once = (
   return wrap;
 };
 
-const unsafeCallback = (
-  // Extract callback function
-  // It's unsafe: may return null, allows multiple calls
-  args // array, arguments
-  // Returns: function, callback or null
-  // Hint: previous name: `common.cbUnsafe` (deprecated)
-  // Hint: another alias: `common.extractCallback` (deprecated)
-) => {
+// Extract callback function
+// It's unsafe: may return null, allows multiple calls
+//   args - array, arguments
+// Returns: function, callback or null
+// Hint: previous name: `common.cbUnsafe` (deprecated)
+// Hint: another alias: `common.extractCallback` (deprecated)
```

Comment:
Also, add some sort of 'deprecated' label?
/cc @SemenchenkoVitaliy @belochub @nechaido @tshemsedinov wdyt?

Reply:
I'm ok with that, but in this case it's just renaming issue and since this particular function is used there is no need for deprecation label. However, I'll consider using another label for such cases. The ones can be `Alias` or `Previous name`. WDYT?